.github/workflows/tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Run tests
         run: bundle exec rspec
       - name: Upload coverage reports to Codecov
-        uses: codecov/codecov-action@v4
```

Comment:
> Tokenless uploading is unsupported. However, PRs made from forks to the upstream public repos will support tokenless (e.g. contributors to OS projects do not need the upstream repo's Codecov token)
https://github.com/codecov/codecov-action?tab=readme-ov-file#breaking-changes

I downgraded codecov-action version from v4 to v3 because the tokenless upload does not work as described in the repository's README.

There's the same issue here.
https://github.com/codecov/codecov-action/issues/1292

How it works is unstable so I downgraded it.

> https://github.com/codecov/codecov-action/issues/1292#issuecomment-1942158455

But it may be because of the rate limit of codecov. 
I will check the result of CI to see if the error was because of the version upgrade.

Reply:
It seems like rolling back to the v3 doesn't work.
https://github.com/meilisearch/meilisearch-rails/actions/runs/8633011381/job/23675900774?pr=351

Since getting back to the v3 doesn't change the error at all, I feel like there's something wrong with the token set in the repository secret.

Reply:
Hi @KatsukiFujimoto thanks for helping, I was able to fix it in the #353 PR :)

The issue was happening because the codecov gem is deprecated, and I should use the CI only.